tools/bpf2c/bpf_code_generator.h

```diff
@@ -216,31 +215,33 @@ class bpf_code_generator
      * @return Vector of section names.
      */
     std::vector<unsafe_string>
-    program_sections();
+    program_sections() const;
 
     /**
      * @brief Parse the eBPF file.
      *
-     * @param[in] program Program to parse.
+     * @param[in, out] program Program to parse.
```

Comment:
Possible mismatch: This says [in,out], but it's a const pointer and SAL is _In_?

Reply:
Yep, updated.